src/modules/creation.js

```diff
@@ -5,15 +5,21 @@ const list = document.querySelector('.todo-list');
 export default function createList(array) {
   list.innerHTML = '';
   for (let i = 0; i < array.length; i += 1) {
-    list.innerHTML += `<li><input class="checkbox" data-id="${array[i].index}" type="checkbox"><input class= "taskEdit" type="text" value ="${array[i].description}"><i class="fa fa-ellipsis-v" aria-hidden="true" ></i></li>`;
+    list.innerHTML += `<li><input class="checkbox" data-id="${array[i].index}" type="checkbox"><input data-id="${array[i].index}" class= "taskEdit" type="text" value ="${array[i].description}"><i class="fa fa-ellipsis-v" aria-hidden="true" ></i></li>`;
   }
-
   const elements = document.getElementsByClassName('checkbox');
+  const edits = document.getElementsByClassName('taskEdit');
 
+  /* eslint-disable no-plusplus */
   for (let i = 0; i < elements.length; i++) {
```

Comment:
- [ ] OPTIONAL: It's a lot more efficient (faster) to use `map` or even `filter` instead of a regular for loop.  Try it! 💪